### DISCUSSION TITLE
Unable to open "Open GitHub Copilot (additional suggestions in separate pane)"

### DISCUSSION THREAD

**Discussion by Yauhenia-Zhdanovich** [community/community#64098] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I have everything installed correctly and activated. I would like to have a tab open to be able to see all the suggestions in a separate pane. However, there is no option to "Open GitHub Copilot" in the toolbar, and I cannot open it with a keyboard shortcut. :(
<img width="665" alt="image" src="https://github.com/community/community/assets/29694535/c0f4399a-83fa-48a0-a41b-2e341bf1ebbf">

extensions version v1.104.341
vscode version 1.81.1

**Comment by christophercouch** [comment, marked as answer]
Hello Yauhenia-Zhdanovich,

It sounds like you're having trouble accessing the separate pane for GitHub Copilot suggestions in VS Code.

1. Ensure that GitHub Copilot is enabled and signed in within VS Code.
2. Try reloading your VS Code window to refresh all extensions.
3. The usual command to open Copilot's suggestions pane is GitHub Copilot: Open Copilot. You can access this by pressing Ctrl+Shift+P (or Cmd+Shift+P on macOS) to open the command palette and then typing in "Copilot".
4. If the command doesn't appear, it's possible there's a conflict with another extension or a corrupted installation. Consider reinstalling the Copilot extension or checking for updates.

If you continue to face issues, it might be helpful to raise a bug report with the GitHub Copilot team for more in-depth assistance.
